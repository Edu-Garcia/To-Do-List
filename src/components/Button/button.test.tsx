import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom';
import { Button } from '.'
import '@testing-library/jest-dom';

describe('Button Test', () => {
  it('snapshot test button component', () => {
    const { container } = render(
      <BrowserRouter>
        <Button>
          Teste
        </Button>
      </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
  });
});