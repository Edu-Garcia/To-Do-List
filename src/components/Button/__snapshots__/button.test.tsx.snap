// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Test snapshot test button component 1`] = `
<div>
  <button
    class="button undefined"
    type="button"
  >
    Teste
  </button>
</div>
`;
